pom.xml

```diff
@@ -38,6 +38,14 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I don`t see all necessary dependencies, there is no checkstyle plugin

Reply:
But I have a checkstyle plagin 😢 And I don't know what dependencies to add :(

Reply:
I see a checkstyle plugin, but better to ask such questions on the platform and tag us, because we do not see the comments on github up until you request another review,
About the dependencies: I do not think we need `spring-boot-starter-data-jpa` so far, right now I see only some hibernate usage, but not any other `spring-data` functionality, maybe that's what @andrii-hoienko mentioned